utils.py

```diff
@@ -398,20 +405,30 @@ def get_time_in_millisecs(datetime_obj):
         datetime_obj: datetime. An object of type datetime.datetime.
 
     Returns:
-        float. This returns the time in the millisecond since the Epoch.
+        float. This returns the time in the milliseconds since the Epoch.
```

Comment:
Drop the 'the' before milliseconds.
Can you edit this comment as well to explain the returned variable? See the below comment regarding it.